src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -609,7 +612,19 @@ private void OnDiscoveryMessageReceived(ITestDiscoveryEventsHandler2 discoveryEv
             OnDiscoveryAbort(discoveryEventsHandler, ex, false);
         }
     }
-
+    private static bool BlameIsEnabled()
```

Comment:
Is the content of this method copy-pasted from somewhere else or is it something you came up with?